tools/list_keyrings.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
-eval g++ $(pkg-config --cflags --libs gnome-keyring-1) list_keyrings.cc -o list.out &&
+cd $(dirname $0)
```

Comment:
Please write `cd $(dirname $0) &&`. Any command fail should fail the whole script.